### PR TITLE
Create public apps with cloud native

### PR DESCRIPTION
**Pull request by alasarr**
It allows to create public apps with cloud native.

If the `oauth`is missing in the initialSlice the app is public and no login form will appear.

How it works:

1. Create a public token from the tokens api
2. Remove the oauth property from initialSlice.
3. Add the token at initialSlice.

```javascript
import { VOYAGER } from '@carto/react-basemaps';
import { API_VERSIONS } from '@deck.gl/carto';

export const initialState = {
  ...
  credentials: {
    apiVersion: API_VERSIONS.V3,
    apiBaseUrl: 'https://gcp-us-east1.api.carto.com',
    accessToken:
      'XXX',
  },
};
```

image

### CODE REVIEW

File: template-skeleton-cn/template/src/routes.js [hunk 1]
```diff
@@ -14,7 +13,6 @@ const routes = [
       // [hygen] Add routes
     ],
   },
-  { path: '/oauthCallback', element: <OAuthCallback /> },
```
Comment (alasarr) [inline template-skeleton-cn/template/src/routes.js:17]: Not part of this PR, but I included here (sorry)

**Review by borja-munoz — COMMENTED** [review]
I like the approach but having a different approach for the skeleton in the current platform and the cloud native platform, makes it more complex to explain how the template works. For instance, here we check the `oauth` property in the `initialState` object but in the current platform we are checking a `forceOauthLogin` property in the appSlice.
It would be easier to explain if we could have the same approach in all the templates.

**Comment by alasarr** [comment]
Not a big deal to change, but have in mind we've already introduced a new change we need to explain because we're not using two slices, that's the reason why I decided to do it like this.

**Comment by alasarr** [comment]
And the parameter forceAuthLogin was introduced because we have an hybrid login (some functionality that requires authentication), it's not supported in cloud native. If we introduce this parameter we should add this too.

**Review by VictorVelarde — APPROVED** [review]
LGTM. I agree on the approach too, so I'm gonna merge it to get an alpha asap

**Review by VictorVelarde — APPROVED** [review]
🚀
